ditto/readers/cyme/read.py

```diff
@@ -4423,6 +4536,22 @@ def parse_transformers(self, model):
             except:
                 pass
 
+            # Set the connection index for the from_element (info is in the section)
+            try:
+                api_transformer["from_element_connection_index"] = int(
+                    self.section_phase_mapping[sectionID]["fromnodeindex"]
+                )
+            except:
+                pass
+
+            # Set the connection index for the from_element (info is in the section)
+            try:
+                api_transformer["to_element_connection_index"] = int(
+                    self.section_phase_mapping[sectionID]["tonodeindex"]
+                )
+            except:
+                pass
+
```

Comment:
All these `try` `except` blocks make me very nervous.

Reply:
Yes we have an active issue to remove these once all the unit tests are in place.